transactional_licence_form/templates/start.html

```diff
@@ -1,4 +1,9 @@
 {% extends "layouts/base.html" %}
+{% block meta_description %}
```

Comment:
I don't think this should be needed if the `context` is set correctly, since the base one should do the job.

Reply:
@jacksonj04 these are only in the static_content not base as all the static pages have extended from static_content not base so I put it this only in this licence page as the static licence pages are not extended from static_content